crowdsec-docs/unversioned/cti_api/taxonomy/false_positives.mdx

```diff
@@ -22,11 +22,21 @@ export const columns = [
 
 <GithubIconRender url={fpURL}></GithubIconRender>
 
+IPs in this category are considered completely safe and trusted. Alerts triggered by these IPs are likely due to misconfiguration or overly sensitive alerting rules.
```

Comment:
Can we tell a bit more about our process of qualifying an IP as SAFE ?
Say that we have a list of genuine know crawlers exposing their IPs and ranges and that we take those in account to qualify an IP as SAFE.

Safe also means that it has no malicious intent, not that it isn't triggering certain scenarios, mainly crawling scenarios